src/amo/sagas/home.js

```diff
@@ -60,121 +53,57 @@ export function* fetchHomeData({
     },
   };
 
+  const recommendedExtensionShelf = (
+    addons: Array<ExternalAddonType>,
+  ): ExternalResultShelfType => {
+    return {
+      title: selectLocalizedContent('Recommended Extensions', state.lang),
+      url:
+        'https://addons.mozilla.org/api/v5/addons/search/?promoted=recommended&sort=random&type=extension',
+      endpoint: 'search',
+      criteria: '?promoted=recommended&sort=random&type=extension',
+      footer: {
+        url: '',
+        outgoing: '',
+        text: selectLocalizedContent(
+          'See more recommended extensions',
+          state.lang,
+        ),
+      },
+      addons,
+    };
+  };
+
   try {
-    let heroShelves = null;
+    let homeShelves = {};
     try {
-      heroShelves = yield call(getHeroShelves, { api: state.api });
+      homeShelves = yield call(getHomeShelves, { api: state.api });
```

Comment:
I don't think we need the hero shelves data when we are on the mobile page, so we can probably avoid calling this API altogether if `isDesktopSite` is `false`. In that case we only need to call the API to retrieve the Recommended Extensions shelf.